modules/zivid/experimental/calibration.py

```diff
@@ -71,6 +72,35 @@ def estimate_intrinsics(frame):
     )
 
 
+def pixel_mapping(camera, settings):
+    """Return pixel mapping information given camera and settings.
+
+    When mapping from a subsampled point cloud to a full resolution
+    2D image it is important to get the pixel mapping correct. This
+    mapping depends on camera model and settings. This function provides
+    the correct parameters to map the 2D coordinates in a point cloud
+    captured using `settings` to the full resolution of the camera.
+
+    Args:
+        camera: Reference to camera instance.
+        settings: Reference to settings instance.
+
+    Returns:
+        A PixelMapping instance.
+    """
+
+    pixel_mapping_handle = _zivid.calibration.pixel_mapping(
+        camera._Camera__impl,  # pylint: disable=protected-access
+        _to_internal_settings(settings),
+    )
+    return PixelMapping(
```

Comment:
I think it is a bit strange that we here have a `_zivid.PixelMapping`, but we don't pass that into the constructor, instead the constructor creates another `_zivid.PixelMapping`.

Reply:
It constructs and returns `zivid.experimental._pixel_mapping` which is mapped to `zivid.experimental.PixelMapping`.

Reply:
There was a chicken and egg story here, right Eskil?

Reply:
The problem was that we really needed two constructors:

- One for the user that takes the four numbers
- One for us that takes a _zivid.PixelMapping

Since Python can't really overload constructors, we opted to only have the first one.
Open to better solutions, but I think this is fine.

Reply:
Yeah, I'm fine with keeping this. We can find a cleaner solution in the future.